packages/magick/install

```diff
@@ -2,9 +2,16 @@
 set -x
 set -e
 
-# PPA has recent ImageMagick versions for for Ubuntu Trusty/Xenial
+OS_DISTRIBUTION=$(lsb_release -cs)
+
 apt-get update -qq
 apt-get install -y software-properties-common python-software-properties
-add-apt-repository -y ppa:opencpu/imagemagick
-apt-get update -qq
+
+# these sysdeps are available by default in bionic
```

Comment:
There is a desire to get the newer versions from the PPA which is why it isn't fetched from the main apt repo.

Reply:
Does not support bionic: https://launchpad.net/~opencpu/+archive/ubuntu/imagemagick

Reply:
True. Hopefully the main repo will be new enough for the folks who care.